src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml.cs

```diff
@@ -85,6 +85,11 @@ private async Task MigrateAsync()
                 logger.LogVerbose(MigrationStrings.CancelTokenFailure_VerboseLog, ex);
                 logger.WriteLine(MigrationStrings.CancelTokenFailure_NormalLog, ex.Message);
             }
+            catch(Exception ex)
+            {
+                logger.LogVerbose(MigrationStrings.ErrorDuringMigation_VerboseLog, ex);
```

Comment:
I really struggle with knowing when to log verbose and when just normal logging is fine so I usually log both since it covers the most scenarios.

Reply:
I think it's a good pattern to follow for errors: the user will normally see the "normal" verbosity message if something goes wrong, and we can ask for the verbose message to help with support/troubleshooting.